boards/adafruit/feather_esp32s3/doc/index.rst

```diff
@@ -0,0 +1,284 @@
+.. _feather_esp32s3:
+
+Adafruit Feather ESP32S3
+########################
+
+Overview
+********
+
+The Adafruit Feather ESP32S3 is an ESP32 development board in the Feather
+standard layout, sharing peripheral placement with other devices labeled as
+Feathers or FeatherWings. The board is equipped with an ESP32-S3 mini module, a
+LiPo battery charger, a fuel gauge, a USB-C and Qwiic/STEMMA-QT connector. For
+more information, check `Adafruit Feather ESP32S3`_
+
+.. figure:: img/adafruit_feather_esp32s3.jpg
+   :width: 640px
+   :align: center
+   :alt: Board Name
+
+   Adafruit Feather ESP32S3 (Credit: Adafruit Industries)
+
+Hardware
+********
+
+- ESP32-S3 mini module, featuring the dual core 32-bit Xtensa Microprocessor
+  (Tensilica LX7), running at up to 240MHz
+- 512KB SRAM and either 8MB flash or 4MB flash + 2MB PSRAM, depending on the
+  module variant
+- USB-C directly connected to the ESP32 for USB/UART and JTAG debugging
+- LiPo connector and built-in battery charging when powered via USB-C
+- MAX17048 fuel gauge for battery voltage and state-of-charge reporting
+- Charging indicator LED, user LED, reset and boot buttons
+- Built-in NeoPixel indicator RGB LED
+- STEMMA QT connector for I2C devices, with switchable power for low-power mode
+
+Asymmetric Multiprocessing (AMP)
+================================
+
+The ESP32-S3 SoC allows 2 different applications to be executed in asymmetric
+multiprocessing. Due to its dual-core architecture, each core can be enabled to
+execute customized tasks in stand-alone mode and/or exchanging data over OpenAMP
+framework. See :ref:`ipc_samples` folder as code reference.
+
+For more information, check the datasheet at `ESP32-S3 Datasheet`_.
+
+Supported Features
+==================
+
+The current Adafruit Feather ESP32S3 board configuration supports the following
```

Comment:
replace with the board name as used for west build, enclosed with \`\` and call it board target